src/components/legend/draw.js

```diff
@@ -63,6 +63,19 @@ module.exports = function draw(gd) {
         .call(Color.fill, opts.bgcolor)
         .style('stroke-width', opts.borderwidth + 'px');
 
+
+    var title = fullLayout.legend.title;
+    gd._fullLayout._legendTitleWidth = 0;
+    gd._fullLayout._legendTitleHeight = 0;
+    if(title.text) {
+        var titleEl = Lib.ensureSingle(legend, 'text', 'legendtitletext');
+        titleEl.attr('text-anchor', 'start')
+            .classed('user-select-none', true)
+            .call(Drawing.font, title.font)
+            .text(title.text);
+
+        textLayout(titleEl, legend, gd); // handle mathjax or multi-line text and compute title height
```

Comment:
Nice!! Looks like you got MathJax to work:

![image](https://user-images.githubusercontent.com/6675409/69647282-319f7180-1037-11ea-86e3-9d4b95713882.png)